scripts/puppeteer.server.js

```diff
@@ -83,7 +83,7 @@ document.getElementById('actions').addEventListener('click', async (event) => {
   try {
     switch (event.target.id) {
       case 'mute-audio':
-        await webexSDKAdapter.meetingsAdapter.meetingControls.audio.action(MEETING_ID);
+        await webexSDKAdapter.meetingsAdapter.meetingControls['mute-audio'].action(MEETING_ID);
```

Comment:
not part of your work but can you make the code look consistent? for both `audio` and `video` cases?